common/src/IO/ObjSerializer.h

```diff
@@ -125,7 +126,7 @@ namespace TrenchBroom {
             std::optional<BrushObject> m_currentBrush;
             std::vector<Object> m_objects;
         public:
-            explicit ObjSerializer(std::ostream& objStream, std::ostream& mtlStream, std::string mtlFilename);
+            explicit ObjSerializer(std::ostream& objStream, std::ostream& mtlStream, const Path& mtlPath);
```

Comment:
`mtlPath` needs to be passed by value, and you can drop `explicit` while you're here.